test/dlc_tests/container_tests/bin/benchmark/get_mxnet_avg_throughput.py

```diff
@@ -0,0 +1,27 @@
+# usage: python <script_name>.py <log_file> 4 Training & Inference GPU
+# usage: python <script_name>.py <log_file> 1 Inference CPU
+import sys
+
+
+def get_avg_speed(filepath, index):
+    total = 0.0
+    n = 0
+    with open(filepath) as f:
+        for line in f:
+            if "Speed" in line:
+                try:
+                    total += float(line.split()[index])
+                except ValueError as e:
+                    raise RuntimeError("LINE: {} split {} ERROR: {}".format(line, line.split()[index], e))
+                n += 1
+    if total and n:
+        return total/n
+    else:
+        raise ValueError("total: {}; n: {} -- something went wrong".format(total, n))
+
+
+if __name__ == '__main__':
+    filepath = sys.argv[1]
+    index = int(sys.argv[2])
```

Comment:
Instead of relying on sys.argv, please make the arguments more explicit by using argparse.

Reply:
Working on it

Reply:
Addressed it in latest revision